DEVELOPMENT.rst

```diff
@@ -166,15 +166,8 @@ Then add database user and database. For Ubuntu 16.04 e. g. run::
     $ sudo -u postgres createdb --owner=openslides openslides
 
 
-2. Install additional packages
```

Comment:
This should be at least somewhere mentioned.

Reply:
See requirements.txt and line 54